front/app/containers/Admin/projects/project/phase/phaseParticipationConfig/components/ParticipationMethodPicker.tsx

```diff
@@ -62,7 +62,9 @@ const ParticipationMethodPicker = ({
   const [methodToChangeTo, setMethodToChangeTo] =
     useState<ParticipationMethod | null>(null);
   const [showSurveyOptions, setShowSurveyOptions] = useState(
-    selectedMethod === 'native_survey'
+    participation_method === 'native_survey' ||
```

Comment:
Fixing a bug where if you select a survey-related participation method the config now works as expected, not only for native surveys.